### PR TITLE
docs: changes to documentation

### PR DESCRIPTION
**Pull request by mohamed-khoualed**
docs: changes to documentation
small grammatical update to the documentation

### CODE REVIEW

File: doc/source/reference/index.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ API reference
 =============
 
 This page gives an overview of all public pandas objects, functions and
-methods. All classes and functions exposed in ``pandas.*`` namespace are public.
+methods. All classes and functions are exposed in ``pandas.*`` namespace are public.
```
Comment (jreback) [inline doc/source/reference/index.rst:10]: revert this as, or add 'the' later in the sentence

File: doc/source/development/contributing.rst [hunk 2]
```diff
@@ -192,7 +192,7 @@ Installing a C compiler
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 pandas uses C extensions (mostly written using Cython) to speed up certain
-operations. To install pandas from source, you need to compile these C
+operations. To install pandas from the source, you need to compile these C
```
Comment (jreback) [inline doc/source/development/contributing.rst:195]: revert this, its a common saying

File: doc/source/development/contributing.rst [hunk 3]
```diff
@@ -145,7 +145,7 @@ as the version number cannot be computed anymore.
 Creating a development environment
 ----------------------------------
 
-To test out code changes, you'll need to build pandas from source, which
+To test out code changes, you'll need to build pandas from the source, which
```
Comment (jreback) [inline doc/source/development/contributing.rst:148]: revert this

**Review by jreback — CHANGES_REQUESTED** [review]

File: doc/source/development/contributing.rst [hunk 4]
```diff
@@ -89,8 +89,8 @@ The issue will then show up to the pandas community and be open to comments/idea
 Working with the code
 =====================
 
-Now that you have an issue you want to fix, enhancement to add, or documentation to improve,
-you need to learn how to work with GitHub and the pandas code base.
+Now that you have an issue you want to fix, an enhancement to add, or documentation to improve,
+you need to learn how to work with GitHub and the pandas codebase.
```
Comment (Bhard27) [inline doc/source/development/contributing.rst:93]: In computer science programming, we usually write Codebase and not Code base/ Code-Base. So I think this is not that necessary.

**Comment by simonjayhawkins** [comment]
@mohamed-khoualed closing as stale. ping when ready to continue

**Pull request closed without merging**